src/Forms/FormAction.php

```diff
@@ -93,9 +97,45 @@ public function getSchemaDataDefaults()
         $defaults = parent::getSchemaDataDefaults();
         $defaults['attributes']['type'] = $this->getUseButtonTag() ? 'button' : 'submit';
         $defaults['data']['icon'] = $this->getIcon();
+        $defaults['data']['savedTitle'] = $this->getSavedTitle();
+        $defaults['data']['savedClasses'] = $this->getSavedClasses();
+        $defaults['data']['savedIcon'] = $this->getSavedIcon();
         return $defaults;
     }
 
+    public function setSavedTitle($title)
```

Comment:
I don't recommend mixing `this.props` and `this.props.data` like that `getProps()` suggestion...
if the data cannot be clearly discovered in a given path and needs to be explored in multiple areas, maybe the API needs to be re-assessed.